package.json

```diff
@@ -12,14 +12,18 @@
     "lint": "pnpm run -r --parallel --aggregate-output lint",
     "format": "pnpm run -r --parallel --aggregate-output format",
     "test": "pnpm run -r --parallel --aggregate-output test",
+    "test-e2e": "pnpm run -r --parallel --aggregate-output test-e2e",
```

Comment:
⚠️ **VERY** important that this is not run as part of `"test"`. We don’t need to run this multiple times in different OSs and and different Node versions. We only need to run this **ONCE** in a single environment (Ubuntu, Node latest because that’s the cheapest/fastest).